dev-requirements-py3.txt

```
@@ -1,14 +1,14 @@
 flake8==3.8.3
 bandit==1.6.2
-demisto-py==2.0.11
+demisto-py>=2.0.11
```

All versions should be locked to avoid unexpected failures. Don't use '>='.
```suggestion
demisto-py==2.0.19
```

---

Done